app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginFragment.java

```diff
@@ -0,0 +1,25 @@
+package ar.com.wolox.android.example.ui.example.login;
+
+import ar.com.wolox.android.R;
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment;
+
+/**
+ * A simple {@link WolmoFragment} subclass.
+ */
+public class LoginFragment extends WolmoFragment<LoginPresenter> implements ILoginView {
+
+    @Override
+    public void init() {
+
+    }
+
+    @Override
+    public int layout() {
+        return R.layout.fragment_login;
+    }
+
+    @Override
+    public void onUsernameSaved() {
+
```

Comment:
Add here a TODO to know which is the purpose of this implementation

Reply:
✔ **Done**